vmdb/db/migrate/20150401090146_add_systemd_and_openstack_related_columns_to_system_services.rb

```diff
@@ -0,0 +1,9 @@
+class AddSystemdAndOpenstackRelatedColumnsToSystemServices < ActiveRecord::Migration
+  def change
+    add_column :system_services, :load, :string
+    add_column :system_services, :active, :string
+    add_column :system_services, :sub, :string
+    add_column :system_services, :openstack_host_service_id, :bigint
+    add_index  :system_services, :openstack_host_service_id
```

Comment:
@roliveri @movitto Can you please review these additions.  I'm not sure on the names of the columns and if they align with the systemd changes that were made recently for fleecing.  I'm also concerned they mess up plans already made with respect to those changes.

Some things that I find weird/potentially bad with this are:
- Naming a column load might be bad as it shadows Ruby's `Kernel#load` method
- Naming a column sub might be bad because it shadows Ruby's `Kernel#sub` method.
- Naming a column active, but not having it be a boolean is confusing.  I would expect it to be boolean, thus creating a `SystemService#active?` method.